doc/requirements/io_class

```diff
@@ -19,65 +19,336 @@ It shall be possible to set a maximum size of an IO class.
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+title: Export an IO class configuration to csv
+id: export_csv
+---
+
+The product shall provide an option to export current IO class configuration to
+CSV format. The exported configuration shall be compatilbe with an config file.
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+title: Load configuration from file
+id: config_file
+---
+
+The product shall provide an option to load IO class configuration from a file.
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+title: Setting an IO class configuration on loading a cache
+id: load_config
+---
+
+When a cache state is loaded, an IO class configuration and all its attributes
+shall be restored to the values used before the cache had been shut down.
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+title: Max number of IO classes
+id: max_configured
+---
+
+The product shall allow to configure up to 33 IO classes.
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+title: Attributes
+id: attributes
+---
+
+The IO class configuration shall include a following attributes for each IO class:
+
+1. Id
+2. Classification rule
+3. Eviction priority
+4. Occupancy limit
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+title: Allowed ids
+id: id_allowed_values
+---
+
+The product shall accept the IO class ids in range 0-32 inclusive.
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+title: Allowed eviction priorities
+id: eviction_priorities_allowed_values
+---
+
+The product shall accept the eviction priorities in range 0-255 inclusive.
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+title: Empty eviction priority
+id: empty_priority
+---
+
+The product shall allow to omit eviction priority during specifying IO class
+configuration.
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+title: Pinning IO class
+id: pinning
+---
+
+The product shall allow to pin IO classes.
```

Comment:
What happens when the IO class is pinned?

Reply:
Agreed, I tnkink "pinning" is never used in user documentation